spec/support/shared_examples/participant_actions_resume_support.rb

```diff
@@ -19,3 +19,79 @@
     expect { described_class.call(params: given_params) }.to raise_error(ActiveRecord::RecordInvalid)
   end
 end
+
+RSpec.shared_examples "JSON Participant Resume endpoint" do |serialiser_type|
+  let(:parsed_response) { JSON.parse(response.body) }
+
+  it "changes the training status of a participant to active" do
+    put url, params: params
+
+    expect(response).to be_successful
+
+    expect(parsed_response.dig("data", "type")).to eq(serialiser_type)
+  end
+
+  it "returns an error when the participant is already active" do
+    2.times { put url, params: params }
+
+    expect(response).not_to be_successful
+  end
+
+  it "returns an error when the participant is withdrawn" do
+    put withdrawal_url, params: withdrawal_params
+    put url, params: params
+
+    expect(response).not_to be_successful
+  end
+end
+
+RSpec.shared_examples "JSON Participant resume documentation" do |url, request_schema_ref, response_schema_ref, tag|
```

Comment:
nitpick, but for these shared examples, you can actually use keyword arguments, eg:
```suggestion
RSpec.shared_examples "JSON Participant resume documentation" do |url: ,request_schema_ref:, response_schema_ref:, tag:|
```
I think it makes it much easier to read since the shared example code won't be next to its usage.